benchmarks/CMakeLists.txt

```diff
@@ -1,63 +1,66 @@
-﻿cmake_minimum_required(VERSION 3.12 FATAL_ERROR)
+﻿cmake_minimum_required(VERSION 3.18 FATAL_ERROR)
 
-project(CUCO_BENCHS LANGUAGES C CXX CUDA)
+project(CUCO_BENCHS LANGUAGES CXX CUDA)
```

Comment:
> I'm curious as to why these were created.

Mainly because I don't know what I'm doing and just copy/paste stuff from other projects :upside_down_face: 

If you remove the `project`, how do you declare what the language requirements are?

Reply:
> just copy/paste stuff from other projects 🙃

That is the typical way to write CMake 😁

> If you remove the `project`, how do you declare what the language requirements are?

They're inherited from the parent project. Typically there's just one cmake project at the root of a repository.